te_markdown/terminology.md

```diff
@@ -8,7 +8,33 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 ## Overview
 
-<!--- TODO: insert (annotated?) overview image and briefly describe the main areas of the user interface -->
+![Overview](/images/te.overview.svg?sanitize=true){:class="img-responsive"}
+
+### The different areas of the Test-Editor-Web
+In the screenshot above you can see the different areas of the Test-Editor-Web which are divided
+through the different yellow areas. The description of the specific area can be found under the number of this area underneath.   
+
+#### 1. The Test-Navigator area
+In this area the user can specify the structure of the different test artifacts.
+He can create, delete, rename, cut, copy and paste test artifacts and refresh the workspace.
+The user can filter the different test-artifacts, so that just the specific test-artifact is visible which was selected.
+
+#### 2. The Editor area
+This area is the place to write different test artifacts. The editor supports the user with a content completion of the different possibilities during typing and saving your test case e.g. through the shortcut ```STRG + S``` . 
+
+#### 3. The Test-Step-Selector area
+The user can find the different testdriver test steps as an overview in the Test-Step-Selektor area, to define test cases.
```

Comment:
I think the user probably does not know about test drivers (and really doesn't have to to understand what the test step selector does).